docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc

```diff
@@ -530,6 +536,10 @@ In this case, the elapsed time of using LOAD Statement without options is nearly
 using LOAD WITH REBUILD INDEXES, and the elapsed time of the former is less than the latter 
 if this CQD is turned ON.
 
+** `UPSERT USING LOAD`
++
+specifies that data is inserted into target tables using rowset inserts without transactions.

```

Comment:
Maybe we could clarify this a bit more? We could mention that in this case we don't use the HBase bulk loader and instead insert the data, multiple rows at a time, using the regular HBase interface (Put).

Reply:
Dank fuer Ihnen hervorragenden Rat. Ich will Ihre Vorschläge günstig aufnehmen. 😁